dll/cpl/sysdm/virtmem.c

```diff
@@ -569,27 +577,45 @@ OnSelChange(HWND hwndDlg, PVIRTMEM pVirtMem)
         }
 
         /* Set minimum pagefile size */
-        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("16 MB"));
+        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("2 MB"));
```

Comment:
Is there a reason to change this ?

Reply:
Yes, in Windows minimal value is 2 MB. No reason to request more with ROS.

Reply:
Maybe our borked memory manager could require more (but I'm not even sure that pagefile actually works in ReactOS). Maybe you can try with this value on ReactOS to see if it doesn't break anything.

Reply:
Of course I did :)